src/drivers/github.go

```diff
@@ -23,7 +24,7 @@ type githubCodeHostingDriver struct {
 }
 
 func (d *githubCodeHostingDriver) CanBeUsed(driverType string) bool {
-	return driverType == "github" || d.hostname == "github.com"
+	return driverType == "github" || d.hostname == "github.com" || d.hostname == os.Getenv("GHE_HOST")
```

Comment:
I think this can be avoided by configuring the code hosting driver referenced here: https://github.com/Originate/git-town/blob/master/documentation/commands/new-pull-request.md#description

Reply:
That's great! I will close this PR.